CHANGELOG.md

```diff
@@ -89,6 +89,9 @@ ___
 ## Unreleased (Master Branch)
 [Full Changelog](https://github.com/parse-community/parse-server/compare/4.5.0...master)
 ### Breaking Changes
+Leveraging database real-time hooks, schema caching has been drastically improved. These improvements allows for reduced calls to the DB, faster queries and prevention of memory leaks. A breaking change can occur if you are horizontally scaling Parse Server (multiple Parse Server instances connecting to the same DB). Set `enableSchemaHooks` parameter in [Parse Server Options](https://parseplatform.org/parse-server/api/master/ParseServerOptions.html) (`enableSingleSchemaCache` and `schemaCacheTTL` have been removed). If you are horizontal scaling instances connected to MongoDB, you must use replica set clusters with WiredTiger, see [ChangeStream](https://docs.mongodb.com/manual/changeStreams/#availability)
+
+The new schema cache uses a singleton object that is stored in-memory. In a horizontally scaled environment, if you update the schema in one instance the DB hooks will update the schema in all other instances. `enableSchemaHooks: true` enables the DB hooks. If you have multiple server instances but `enableSchemaHooks: false`, your schema maybe out of sync in your instances (resyncing will happen if an instance restarts). (Diamond Lewis, SebC) [#7214](https://github.com/parse-community/parse-server/issues/7214)
```

Comment:
I think this is a great explanation, should we move that into the Parse Server docs instead and just link to it in the change log?

Reply:
I want developers to clearly see this. It could get easily overlook. Someone like me who has horizontally scaled 8 Parse Server instances should know this.

Reply:
Sure, then I suggest to maybe make this info available in the docs too, in a separate docs PR.